api/src/lib.rs

```diff
@@ -1269,58 +1307,58 @@ pub extern "system" fn Java_com_horizen_merkletreenative_InMemoryOptimizedMerkle
-
-    // Create new BigMerkleTree Rust side
-    let mt = get_ginger_smt(
-        _height as usize,
-        db_path.to_str().unwrap(),
-    ).expect("Should be able to create new BigMerkleTree");
-
-    // Create and return new BigMerkleTree Java side
-
-    let mt_ptr: jlong = jlong::from(Box::into_raw(Box::new(mt)) as i64);
-
-    _env.new_object(_class, "(J)V", &[JValue::Long(mt_ptr)])
-        .expect("Should be able to create new BigMerkleTree object")
-        .into_inner()
-}
-
-#[no_mangle]
-pub extern "system" fn Java_com_horizen_merkletreenative_BigMerkleTree_nativeGetPosition(
-    _env: JNIEnv,
-    _tree: JObject,
-    _leaf: JObject,
-) -> jlong
-{
-    let leaf = {
-
-        let fe =_env.get_field(_leaf, "fieldElementPointer", "J")
-            .expect("Should be able to get field fieldElementPointer");
-
-        read_raw_pointer(fe.j().unwrap() as *const FieldElement)
-    };
-
-    let tree = {
-
-        let t =_env.get_field(_tree, "merkleTreePointer", "J")
-            .expect("Should be able to get field merkleTreePointer");
-
-        read_raw_pointer(t.j().unwrap() as *const GingerSMT)
-    };
-
-    get_position_in_ginger_smt(tree, leaf) as jlong
-}
+// #[no_mangle]
+// pub extern "system" fn Java_com_horizen_merkletreenative_BigMerkleTree_nativeInit(
+//     _env: JNIEnv,
+//     _class: JClass,
+//     _height: jint,
+//     _db_path: JString,
+// ) -> jobject
+// {
+//     // Read db_path
+//     let db_path = _env.get_string(_db_path)
+//         .expect("Should be able to read jstring as Rust String");
+//
+//     // Create new BigMerkleTree Rust side
+//     let mt = get_ginger_smt(
+//         _height as usize,
+//         db_path.to_str().unwrap(),
+//     ).expect("Should be able to create new BigMerkleTree");
+//
+//     // Create and return new BigMerkleTree Java side
+//
+//     let mt_ptr: jlong = jlong::from(Box::into_raw(Box::new(mt)) as i64);
+//
+//     _env.new_object(_class, "(J)V", &[JValue::Long(mt_ptr)])
+//         .expect("Should be able to create new BigMerkleTree object")
+//         .into_inner()
+// }
+
+// #[no_mangle]
+// pub extern "system" fn Java_com_horizen_merkletreenative_BigMerkleTree_nativeGetPosition(
+//     _env: JNIEnv,
+//     _tree: JObject,
+//     _leaf: JObject,
+// ) -> jlong
+// {
+//     let leaf = {
+//
+//         let fe =_env.get_field(_leaf, "fieldElementPointer", "J")
+//             .expect("Should be able to get field fieldElementPointer");
+//
+//         read_raw_pointer(fe.j().unwrap() as *const FieldElement)
+//     };
+//
+//     let tree = {
+//
+//         let t =_env.get_field(_tree, "merkleTreePointer", "J")
+//             .expect("Should be able to get field merkleTreePointer");
+//
+//         read_raw_pointer(t.j().unwrap() as *const GingerSMT)
+//     };
+//
+//     get_position_in_ginger_smt(tree, leaf) as jlong
+// }
 
 #[no_mangle]
 pub extern "system" fn Java_com_horizen_merkletreenative_BigMerkleTree_nativeGetAbsolutePosition(
```

Comment:
This is not using smt but I think it's not needed too

Reply:
Agree, let's comment all `com.horizen.merkletreenative` package classes related JNIs.

Reply:
Done